plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/BaseClickHouseTypeMapping.java

```diff
@@ -386,7 +386,7 @@ public void testDouble()
         SqlDataTypeTest.create()
                 .addRoundTrip("double", "3.1415926835", DOUBLE, "DOUBLE '3.1415926835'")
                 .addRoundTrip("double", "1.79769E308", DOUBLE, "DOUBLE '1.79769E308'")
-                .addRoundTrip("double", "2.225E-307", DOUBLE, "DOUBLE '2.225E-307'")
+                .addRoundTrip("double", "2.2250738585072014E-308", DOUBLE, "DOUBLE '2.2250738585072014E-308'")
```

Comment:
Why did you change this value?

Reply:
there were too many merge conflicts/that branch was stale. adding the same changes into this PR

Reply:
somethings changed in clickhouse 22.8 -> 22.9. i am unable to point at what. pasting the 22.8 and 22.9 results for the same 3 queries

```sql
 create table myTable( col1 Float32, col2 Float64) ENGINE=MergeTree() ORDER BY col1
insert into myTable VALUES (2.225E-307, 2.225E-307);
select * from myTable
```

**22.9**
![image](https://github.com/trinodb/trino/assets/126167264/b4000b84-c487-4053-80a5-ca1d07fcd651)



**22.8**
![image](https://github.com/trinodb/trino/assets/126167264/2f1d1dd8-ea55-4095-9252-e1a124d2f81d)

Reply:
Please file an issue to ClickHouse and leave the link here. 

Also, please add a test verifying the correctness instead of removing the test so that we can catch the change of ClickHouse.

Reply:
https://github.com/ClickHouse/ClickHouse/issues/56381.

@ebyhr i have added the testcases that will catch a fix(once made my clickhouse)